netplugin/netd.go

```diff
@@ -337,7 +337,7 @@ func handleContainerStart(netPlugin *plugin.NetPlugin, crt *crt.Crt,
 
 	err = attachContainer(netPlugin.StateDriver, crt, contName)
 	if err != nil {
-		log.Printf("error attaching container err \n", err)
+		log.Print("error attaching container err \n", err)
```

Comment:
Minor: Should change the above to print the error i.e. move the string to `"error %s attaching..."` and continue using Printf here. Most of the errors are printed that way.